gr-digital/python/digital/qa_constellation_encoder_bc.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+#
+# Copyright 2004,2007,2010-2013,2020 Free Software Foundation, Inc.
+#
+# This file is part of GNU Radio
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+#
+
+
+from gnuradio import gr, gr_unittest, digital, blocks
+
+
+class test_constellation_encoder(gr_unittest.TestCase):
```

Comment:
There are only tests with `d_dim == 1`. A test with a different value would be valuable.

Reply:
Agreed with @jdemel here. I don't think we test for `dim > 1` anywhere. Be useful here as well as in the decoder. Shouldn't be difficult to implement I think ... 🤞 that it works without much effort!